lib/locations/location.ex

```diff
@@ -1,19 +1,42 @@
-defmodule Locations.Location do
-  defstruct vehicle_id: nil,
-            status: nil,
-            stop_id: nil,
-            timestamp: nil,
-            route_id: nil,
-            trip_id: nil,
-            consist: []
+defmodule Locations do
+  defmodule CarriageDetails do
```

Comment:
There's a pretty strong pattern of one-module-per-file in non-test code. Perhaps this should live in a new `locations/carriage_details.ex` file?